oracle/oracle.go

```diff
@@ -259,3 +264,13 @@ func (o *OracleImpl) GetPrices() types.Prices {
 	prices := o.priceAggregator.GetPrices()
 	return prices
 }
+
+// GetMarketMap returns the current market map configuration.
+//
+// TODO(Tyler): is the empty map ok? maybe? maybe not. lord save us all.
+func (o *OracleImpl) GetMarketMap() marketmaptypes.MarketMap {
+	if o.marketMapGetter != nil {
+		return o.marketMapGetter()
+	}
+	return marketmaptypes.MarketMap{}
+}
```

Comment:
if no option is passed in, ok to just empty return?

Reply:
im cool w that

Reply:
Alternatively, you can pass the `getMarketMapFn` closure as an argument to construct the `Oracle`, and fail the constructor if it's nil. Either way works IMO

Reply:
totally -- just wasn't sure how delicate we wanted to be w/ breaking the constructor signature. 

or perhaps i should default to the static config.. cause theres _always_ a marketmap, right? this endpoint is just to view it as it changes, so maybe serving whatever default (if there is one) would be nicer here..

Reply:
gonna mark this as resolved as this will be a tad different after BLO-1329